examples/patterns/navigation/subnav.html

```diff
@@ -0,0 +1,145 @@
+    </span>
+    <ul class="p-navigation__links" role="menu">
+      <li class="p-navigation__link p-subnav" role="menuitem" id="link-1">
+        <a href="#link-1-menu" class="p-subnav__toggle">Link 1</i>
+        </a>
+        <ul class="p-subnav__items" id="link-1-menu" aria-hidden="true" style="display: none;">
+          <li>
+            <a href="#" class="p-subnav__item">Sub item 1</a>
+          </li>
+          <li>
+            <a href="#" class="p-subnav__item">Sub item 2</a>
+          </li>
+          <li>
+            <a href="#" class="p-subnav__item">Sub item 3</a>
+          </li>
+        </ul>
+      </li>
+      <li class="p-navigation__link p-subnav" role="menuitem" id="link-2">
+        <a href="#link-2-menu" class="p-subnav__toggle">Link 2</i>
+        </a>
+        <ul class="p-subnav__items" id="link-2-menu" aria-hidden="true" style="display: none;">
+          <li>
+            <a href="#" class="p-subnav__item">Sub item 1</a>
+          </li>
+          <li>
+            <a href="#" class="p-subnav__item">Sub item 2</a>
+          </li>
+          <li>
+            <a href="#" class="p-subnav__item">Sub item 3</a>
+          </li>
+          <li>
+            <a href="#" class="p-subnav__item">Sub item 4</a>
+          </li>
+          <li>
+            <a href="#" class="p-subnav__item">Sub item 5</a>
+          </li>
+        </ul>
+      </li>
+      <li class="p-navigation__link p-subnav" role="menuitem" id="link-3">
+        <a href="#link-3-menu" class="p-subnav__toggle">Link 3</a>
+        <ul class="p-subnav__items" id="link-3-menu" aria-hidden="true" style="display: none;">
+          <li>
+            <a href="#" class="p-subnav__item">Sub item 1</a>
+          </li>
+          <li>
+            <a href="#" class="p-subnav__item">Sub item 2</a>
+          </li>
+          <li>
+            <a href="#" class="p-subnav__item">Sub item 3</a>
+          </li>
+        </ul>
+      </li>
+    </ul>
+  </nav>
+</header>
+
+<script>
+var dropdowns = document.querySelectorAll('.p-subnav__items');
+var dropdownToggles = document.querySelectorAll('.p-subnav');
+var dropdownToggleLinks = document.querySelectorAll('.p-subnav__toggle');
+
+dropdownToggleLinks.forEach(function(dropdownToggleLink) {
+  dropdownToggleLink.addEventListener('click', function(event) {
+    event.preventDefault();
+  });
+});
+
+dropdownToggles.forEach(function(dropdownToggle) {
+  dropdownToggle.addEventListener('click', function(event) {
+    event.stopPropagation();
+
+    var clickedDropdown = this;
+
+    dropdownToggles.forEach(function(dropdownToggle) {
+      var dropdownContent = dropdownToggle.querySelector('.p-subnav__items');
+      if (dropdownToggle === clickedDropdown) {
+        if (dropdownToggle.classList.contains('is-active')) {
+          closeMenu(dropdownToggle, dropdownContent);
+        } else {
+          dropdownToggle.classList.add('is-active');
+          showElement(dropdownContent);
```

Comment:
If you have a closeMenu function it makes sense to move this to a openMenu.

Reply:
done